catalyst_voices/packages/libs/catalyst_cardano_serialization/test/builders/input_builder_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:kiri_check/kiri_check.dart';
+import 'package:test/test.dart';
+
+import '../test_utils/selection_utils.dart';
+
+void main() {
+  const config = SelectionUtils.defaultConfig;
+  final changeAddress = SelectionUtils.randomAddress();
+
+  group('TransactionBuilder Coin Selection Tests', () {
```

Comment:
Often when referring to classes we can inline the type so that when the class is renamed the test is also updated.
```suggestion
  group('$TransactionBuilder Coin Selection Tests', () {
```